spec/models/footer_spec.rb

```diff
@@ -0,0 +1,32 @@
+require "rails_helper"
+
+RSpec.describe Footer, type: :model do
+  before {
+    allow(Rails).to receive(:root).and_return(file_fixture("_settings").join("..").cleanpath)
```

Comment:
[suggestion] I don't think you need to do this, since you are passing the filepath directly in the test.